test/functional/wallet_orphanedreward.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020-2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test orphaned block rewards in the wallet."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class OrphanedBlockRewardTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def run_test(self):
+        # Generate some blocks and obtain some coins on node 0.  We send
+        # some balance to node 1, which will hold it as a single coin.
+        self.nodes[0].generate(150)
+        self.nodes[0].sendtoaddress(self.nodes[1].getnewaddress(), 10)
+        self.nodes[0].generate(1)
+
+        # Get a block reward with node 1 and remember the block so we can orphan
```

Comment:
```suggestion
        # Get a block reward (25) with node 1 and remember the block so we can orphan
```
(Non-blocking, only if you touch-up, would have made the test easier for me to understand.)